Signal/src/AppDelegate.m

```diff
@@ -219,8 +219,6 @@ - (void)ensureIsReadyForAppExtensions
     [TSStorageManager migrateToSharedData];
     [OWSProfileManager migrateToSharedData];
     [TSAttachmentStream migrateToSharedData];
-
-    [OWSPreferences setIsReadyForAppExtensions:YES];
```

Comment:
I'm moving this flag to until later in the main app launch sequence, since we'll soon be adding additional steps, e.g. "decompose primary storage."